apps/docusaurus/docs/create-aptos-dapp/templates/digital-asset.md

```diff
@@ -190,3 +190,53 @@ Creating a collection on mainnet is the same flow as creating on testnet, but we
 1. Run `npm run build` to build a static site
 2. Run `npm run preview` to see how your dapp would look like on a live server
 3. Next, all you need is to deploy your static site to a live server, there are some options for you to choose from and can follow [this guide](https://vitejs.dev/guide/static-deploy#github-pages) on how to use each
+
+## Update the look and feel of the dapp
```

Comment:
can we keep "Update the look and feel of the dapp" in this page, and move all the others to the FAQ page? can link from here to there.

Reply:
Updated